docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+
+services:
+  meter:
```

Comment:
Would be nice to add the environment variables here as well that can be used to configure Influx connection details.

Reply:
true